bundles/CoreBundle/src/Command/Migrate/StorageCommand.php

```diff
@@ -100,6 +104,11 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         $this->io->success('Finished Migrating Storage!');
 
+        if(!empty($errors)) {
```

Comment:
$errors variable is always set.
```suggestion
        if ($errors) {
```